docs/build.md

```diff
@@ -136,23 +143,38 @@ All commands must be run in the root of the `unicodetools` repository local work
+  * In-source build: `MAVEN_OPTS="-ea" mvn exec:java -Dexec.mainClass="org.unicode.text.UCD.Main"  -Dexec.args="version 14.0.0 build MakeUnicodeFiles"  -pl unicodetools  -DCLDR_DIR=$(cd ../cldr ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd Generated; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=14.0.0`
 
 - Build and Test:
   * Out-of-source build: `MAVEN_OPTS="-ea" mvn package -DCLDR_DIR=$(cd ../../../cldr/mine/src ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd ../Generated ; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=14.0.0`
-  * In-source build: `MAVEN_OPTS="-ea" mvn package -DCLDR_DIR=$(cd ../cldr ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd output/Generated; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=14.0.0`
+  * In-source build: `MAVEN_OPTS="-ea" mvn package -DCLDR_DIR=$(cd ../cldr ; pwd)  -DUNICODETOOLS_GEN_DIR=$(cd Generated; pwd)  -DUNICODETOOLS_REPO_DIR=$(pwd)  -DUVERSION=14.0.0`
+
+#### Running commands in Eclipse
+
+For each individual command in Unicode Tools described above, you can configure a Launch Configuration in one of two ways.
+
+1.  Just like the Build and Test run config described above, which uses Maven, with the following command and extra changes:
+    1. From Run > Run Configurations ..., select the previous "Build and Test" configuration. Then select the "Duplicate" button above to create a new duplicate run config. Now make the following changes.
+    2. Name: [command name goes here]  (ex: `UCD Make Unicode Files`)
+    3. Main > Goals: `-pl unicodetools compile exec:java` (the argument for the subproject list flag `-pl` assumes that the class with the main method is in the subdirectory `unicodetools/src/main/java`)
+    4. In the environment variables section, also set the class containing the main method and the command's CLI args (ex: name = `exec.mainClass`, value = `"org.unicode.text.UCD.Main"`; name = `exec.args`, value = `"version 15.0.0 build MakeUnicodeFiles"`)
+2. Create a typical Eclipse run configuration for running a Java class with a main file
+    1. Run > Run Configurations ... > Java Application, then click the New Launch Configuration icon above
+    2. Name: [command name goes here]  (ex: `UCD Make Unicode Files`)
+    3. Project: `unicodetools`
+    4. Main class:  [main class path]  (ex: `org.unicode.text.UCD.Main`)
+    5. Arguments > Program arguments: [main class args] (ex: `version 15.0.0 build MakeUnicodeFiles`)
+    6. Arguments > VM arguments: [any VM arguments] (ex: `-ea`)
+    7. Environment > Add... . For each environment variable, add an entry, similar as above for the Build and Test run config. (Example: name = `CLDR_DIR`, value = `/Users/echeran/oss/cldr`; name = `UNICODETOOLS_GEN_DIR`, value = `/Users/echeran/oss/unicodetools/Generated`; name = `UNICODETOOLS_REPO_DIR`, value = `/Users/echeran/oss/unicodetools`; name = `UVERSION`, value = `15.0.0`)
```

Comment:
FYI -- Yes, we can do this in each run configuration, but we should recommend setting the variables in the JRE preferences. Otherwise, when something changes (new variable, new version, ...) one would have to remember to make the same change in each run configuration.

Reply:
Done.